index.js

```diff
@@ -2,188 +2,202 @@
 
 const fs = require('fs')
 const path = require('path')
-const steed = require('steed')
```

Comment:
should we remove this from the dependencies then?

Reply:
I removed it.